### ISSUE TITLE
Request for Assistance Regarding Attention Mask and Pad Token ID Warning Affecting Model Performance

### ISSUE THREAD

**Issue by AlexandraDobrescu** [Ucas-HaoranWei/GOT-OCR2.0#97]
I encountered the following warning while running the model:
"The attention mask and the pad token ID were not set. As a result, you might experience unexpected behavior. To ensure reliable results, please provide an attention_mask with your input. The pad_token_id has been set to eos_token_id: 151643 for open-end generation."
I suspect this may be impacting the model's performance. Could you offer guidance on how to address this issue? Any assistance would be greatly appreciated.

**Comment by TaoSeekAI** [comment]
Same issue 
Any ideas ？

**Comment by tairenchen** [comment]
Same problem, any solutions?

**Comment by LifeRainSunshine** [comment]
一样的问题  求解决

**Comment by 450711420** [comment]
same problem please share the solution if u guys solve this already thanks :)

**Comment by KlausFan** [comment]
same problem
![image](https://github.com/user-attachments/assets/14fd4e09-9d98-464f-b301-86c6e4acdd9a)

**Comment by hl-gl** [comment]
> 我认为这是因为 tokenizer 版本。我的 tokenizer 版本是 0.15.2。

我的也是这个版本，但是报错同上面一样
